data/governments.txt

```diff
@@ -786,11 +867,96 @@ government "Korath (Civilian)"
 	language "Korath"
 
 	"attitude toward"
-		"Kor Mereti" -.01
-		"Kor Sestor" -.01
 		"Korath" .01
+		"Independent (Korath-Allied)" .2
+		"Pug (Wanderer)" -.01
+	"penalty for"
+		assist -0.25
+		disable 0.26
+		board 1
+		capture 10
 
 	"player reputation" 1
+	"bribe" .2
+	"send untranslated hails"
+	"friendly hail" "friendly korath untranslated"
+	"friendly disabled hail" "friendly disabled korath untranslated"
+	"hostile hail" "hostile korath civilian untranslated"
+	"hostile disabled hail" "hostile disabled korath untranslated"
+
+government "Korath (Station)"
+	"display name" "Korath"
+	color .8 .5 .1
+	language "Korath"
+
+	"attitude toward"
+		"Wanderer" -.01
+		"Remnant" -.01
+		"Pug (Wanderer)" -.01
+		"Remnant (Research)" -.01
```

Comment:
```suggestion
```

Reply:
The Remnant Research "government" is primarily for specific Remnant ships that the Remnant want the player to scan. In other words, it's not a separate government, faction, or group, it's just a differentiator for "the Remnant won't get upset if you scan this specific ship."

In all honesty, if it was possible to suppress certain aspects of government penalties on a per-ship basis, then this government wouldn't exist at all, and it'd just be the "Winds of Light" and the "Shattered Light" that have the "provoked on scan" suppressed.

Which is all a longwinded way of saying that any change or reputation modification that happens to the Remnant should happen to the Remnant Research; but given that there's all of two ships and no fleets in the game that have this government (and neither of those are combat missions, or involve the Korath at all), you could also just ignore it entirely and it should have no impact on anything.

Reply:
Shattered Light 4 features Korath ships, though, by that point, you are not required to scan the Shattered Light so its government could bew changed to the normal Remnant one.

Reply:
This is the station government, though.